kong/core/plugins_iterator.lua

```diff
@@ -52,18 +52,16 @@ local function iter_plugins_for_req(loaded_plugins, is_access_or_certificate_con
   end
 
   local i = 0
-  local function get_next_plugin()
-    i = i + 1
-    return loaded_plugins[i]
-  end
+  local empty = {}
```

Comment:
We could just create it on the chunk level, to avoid creating a new table on each new iterator.